chall02/edramire.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
```

Comment:
Good 👍But it's always better to specified the version, here we want `python3` as a minimum version. I recommend you to read some documentation about this, there a lot of debates about shebang